switchsai/src/saivlan.c

```diff
@@ -57,6 +58,38 @@ sai_status_t sai_create_vlan_entry(
         switch_status = switch_api_vlan_mld_snooping_enabled_set(
             vlan_handle, true);
         assert(switch_status == SWITCH_STATUS_SUCCESS);
+
+        /* register rx net filter for this vlan */
```

Comment:
What do you think? I could move it up, but it makes the SAI stuff in FBOSS less portable. But maybe it already isn't portable...